poetry.lock

```diff
@@ -502,14 +518,14 @@ numba = ">=0.60,<0.62"
 numpy = ">=1.24"
 pandas = ">=2.2"
 psutil = "*"
-pyarrow = ">=19.0,<19.1"
```

Comment:
we're now pulling in pyarrow 21, which caused the CI issue. likely with how the arrow schema is displayed